Source/CsDebugScript.Engine/Engine/IDebuggerEngine.cs

```diff
@@ -288,5 +288,38 @@ public interface IDebuggerEngine
         /// </summary>
         /// <returns></returns>
         DebugEventInfo GetLastEventInfo();
+
+        /// <summary>
+        /// When doing live process debugging continues debugee execution of the current process.
+        /// </summary>
+        void ContinueExecution();
```

Comment:
Done. For now I left default interface which accepts Process as argument. We should agree on whether we want to have default to be just continue/break without arguments since it seems like these commands are process agnostic.

---
In reply to: [69979453](https://github.com/southpolenator/WinDbgCs/pull/8#discussion_r69979453) [](ancestors = 69979453)